packages/address/README.md

```diff
@@ -147,9 +147,9 @@ If your component uses this package and you want to test it with mock API calls
 
 ```
 import {fetch} from '@shopify/jest-dom-mocks';
-import {mockCountryRequests} from '@shopify/address/tests';
+import {testUtilities} from '@shopify/address';
```

Comment:
is it possible you could have a custom tsconfig just for your test utilities that just compiles straight to the folder? then you could import this like

```
import testUtilities from '@shopify/address/test-utilities';
```